src/Resources/ResourceManager/Implementation/ResourceGroupDeployments/NewAzureResourceGroupDeploymentCmdlet.cs

```diff
@@ -78,6 +78,10 @@ public class NewAzureResourceGroupDeploymentCmdlet : DeploymentCreateCmdlet
         [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
         public SwitchParameter AsJob { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Aux tenant ids for cross tenant deployments")]
+        public string[] AuxTenants { get; set; } 
```

Comment:
```suggestion
        public string[] AuxTenant { get; set; } 
```
For our design guide, please use singular name